backends/v3/src/main.rs

```diff
@@ -68,10 +68,8 @@ struct Args {
     disable_grammar_support: bool,
     #[clap(default_value = "4", long, env)]
     max_client_batch_size: usize,
-    #[clap(long, env, default_value_t)]
-    disable_usage_stats: bool,
-    #[clap(long, env, default_value_t)]
-    disable_crash_reports: bool,
+    #[clap(default_value = "on", long, env)]
+    usage_stats: Option<usage_stats::UsageStatsLevel>,
```

Comment:
Since you're defining a default (And you can define it on the enum) you can skip the `Option` here.

Reply:
With the enum default I think you can even skip the value in clap `#[clap(default, long env)]`

Reply:
True, I'll delete the option from `launcher/src/main.rs` as well.

With `#[clap(default, long env)]` I got a compiler error, `attribute `default` requires a value`.
So seems like it's not supported.